src/OrbitGl/ImGuiOrbit.cpp

```diff
@@ -327,6 +327,7 @@ void SetupImGuiStyle(bool bStyleDark_, float alpha_) {
     }
   }
 }
+}  // namespace
 
 void Orbit_ImGui_RenderDrawLists(ImDrawData* draw_data) {
```

Comment:
This change is probably difficult to understand in the diff view, so let me quickly explain it:
I removed `Orbit_ImGui_RenderDrawLists` from the anonymous namespace by ending the
namespace early (see above). Afterwards I declared the one function below static.

I could of course also move the function to a different place in the file, but that felt weird since
I didn't make any changes to the code.